build/debian/changelog

```diff
@@ -1,3 +1,9 @@
+mixxx (2.1.6-0ubuntu1) UNRELEASED; urgency=medium
+
+  * New upstream release
+
+ -- Daniel <daniel@daniel-530U3C-530U4C-532U3C>  Sun, 23 Dec 2018 11:01:23 +0100
```

Comment:
instead of `daniel@daniel-530U3C-530U4C-532U3C` want to use your `mixxx.org` email?